stdlib/private/StdlibUnittest/StdlibUnittest.swift.gyb

```diff
@@ -16,7 +16,7 @@ import SwiftPrivateDarwinExtras
 
 #if os(OSX) || os(iOS) || os(watchOS) || os(tvOS)
 import Darwin
-#elseif os(Linux)
+#elseif os(Linux) || os(FreeBSD)
 import Glibc
```

Comment:
I'm OK with this in the short term, but as we discussed before, Glibc is not the right module name on FreeBSD.